### ISSUE TITLE
Migrate to manifest v3

### ISSUE THREAD

**Issue by wilkerlucio** [wilkerlucio/media-looper#8]
https://developer.chrome.com/docs/extensions/mv3/intro/mv3-migration/

**Comment by wilkerlucio** [comment]
I'm having problems loading the scripts during development when using manifest v3.

```json
  "web_accessible_resources": [
    {
      "resources":  ["js/*"],
      "matches": ["*://*.youtube.com/*"]
    }
  ]
```

<img width="815" alt="Screen Shot 2021-06-30 at 00 52 43" src="https://user-images.githubusercontent.com/25736/123899395-7ee37380-d93d-11eb-9f93-4e04a94824fe.png">

**Comment by wilkerlucio** [comment]
Closed by 658894630fe2ef10fb6a34a5fd8a84026b1eaacf
